help_to_heat/frontdoor/mock_epc_api.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+import pathlib
+from http import HTTPStatus
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+__here__ = pathlib.Path(__file__).parent
+DATA_DIR = __here__ / "mock_epc_api_data"
+
+
+class MockEPCApi:
+    files = {
+        "address_and_rrn": "sample_address_and_rrn_response.json",
+        "epc": "sample_epc_response.json",
+        "access_token": "sample_token_response.json",
+    }
+
+    def __init__(self, token):
+        self.token = token
+
+    def get_address_and_rrn(self, building, postcode):
+        content = (DATA_DIR / MockEPCApi.files["address_and_rrn"]).read_text()
+        return json.loads(content)
```

Comment:
I've added one in but just left it at the top-level, underneath where `DATA_DIR` is defined. I think this makes sense since it's a utility method, but I could be wrong!

Reply:
That sounds right to me. 👍